lib/config/prettier.js

```diff
@@ -24,4 +24,21 @@ module.exports = {
     // https://github.com/prettier/eslint-config-prettier/blob/5399175c37466747aae9d407021dffec2c169c8b/README.md#no-unexpected-multiline
     'no-unexpected-multiline': 'error',
   }),
+
+  overrides: [
+    {
+      // force babylon parser for graphql files
+      files: ['*.graphql', '*.gql'],
+      rules: {
+        'prettier/prettier': 'off',
+      },
+    },
+    {
+      // force typescript parser for TypeScript files
```

Comment:
This shouldn't be needed as eslint-plugin-prettier 2.7.0 (and really anything after 2.4.0) shall infer a parser based upon the the file extension - including using the typescript parser for ts/tsx files